src/applications/claims-status/containers/ClaimEstimationPage.jsx

```diff
@@ -2,24 +2,37 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { Link } from 'react-router';
 import AskVAQuestions from '../components/AskVAQuestions';
-import Breadcrumbs from '../components/Breadcrumbs';
+import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
 import { setUpPage } from '../utils/page';
-import { getClaimType } from '../utils/helpers';
 
 class ClaimEstimationPage extends React.Component {
   componentDidMount() {
     document.title = 'How We Come Up with Your Estimated Decision Date';
     setUpPage();
   }
+
+  renderBreadcrumbs(claim) {
+    const crumbs = [
+      <a href="/" key="home">Home</a>,
+      <a href="/disability-benefits/" key="disability-benefits">Disability Benefits</a>,
+      <Link to="/" key="claims-home">Track Your Claims and Appeals</Link>,
+      <Link to={`/your-claims/${claim.id}`} key="claim-id">Status Details</Link>,
+      <Link to={`/your-claims/${claim.id}/claim-estimate`} key="claim-id">Estimated Decision Date</Link>
+    ];
+
+    return crumbs;
+  }
+
+
   render() {
-    const claimType = !this.props.loading ? getClaimType(this.props.claim) : '';
+    // const claimType = !this.props.loading ? getClaimType(this.props.claim) : '';
+    const claim = this.props.claim;
     return (
```

Comment:
Is `claimType` not necessary here anymore? Asking because I see the check for `this.props.loading` removed. Have you tested to see what happens when you navigate to this page directly though the address? That might be where the `loading` prop comes in to play.

Reply:
@ncksllvn I loaded this page directly a few times, and the claimType was just updating the breadcrumb link name. Since we've simplified it to `Status Details` on all claim and appeals detail views, I feel okay removing it.